src/schema/rules/hints.yaml

```
@@ -0,0 +1,22 @@
+---
+SuspiciouslyLongDesign:
+  code: SUSPICIOUSLY_LONG_EVENT_DESIGN
+  message: |
+    The onset of the last event is after the total duration of the corresponding scan.
+    This design is suspiciously long.
+  selectors:
+    - suffix == "bold"
+    - associations contains "events"
+  checks:
+    - associations.events.onset[-1] > nifti_header.pixdim[4] * nifti_header.dim[4]
```

I believe this is how we would do this in [CEL](https://github.com/google/cel-spec/blob/master/doc/langdef.md).

```suggestion
    - associations.events.onset.exists(t, t > nifti_header.pixdim[4] * nifti_header.dim[4])
```